src/qml/qgismobileapp.qml

```diff
@@ -1249,6 +1249,7 @@ ApplicationWindow {
     spacing: 4
 
     QfToolButtonDrawer {
+      id: mainToolbarDrawer
```

Comment:
Is that change necessary?

Reply:
Not required, tested some things. IMO it does not hurt, but will remove it upon your request.